### ISSUE TITLE
.properties should return compacted properties

### ISSUE THREAD

**Issue by matthieu-fesselier** [solid/query-ldflex#39]
When using the function `properties` on a resource or a container, I get a list of expanded properties ([example](https://solid.github.io/ldflex-playground/#%5Bhttps%3A%2F%2Fjairocr.inrupt.net%2Fpublic%2Fform.ttl%23addressesComment%5D.properties))

Shouldn't it be compacted? It seems like LDFlex is doing it, and we already have the `predicates` function which returns expanded properties ([example](https://solid.github.io/ldflex-playground/#%5Bhttps%3A%2F%2Fjairocr.inrupt.net%2Fpublic%2Fform.ttl%23addressesComment%5D.predicates))

**Comment by RubenVerborgh** [comment]
Indeed; this depended on https://github.com/rubensworks/jsonld-context-parser.js/issues/21, which now seems to be done. Upgrading the JSON-LD context parser should do the trick.
